### ISSUE TITLE
FastBoot service request info is uninitialized

### ISSUE THREAD

**Issue by zeppelin** [embermap/ember-cli-fastboot-testing#1]
The current value of the request object is the below. This breaks tests written against apps that rely on any of this data. For example, mine needs the request path to determine the locale.

```js
request:
  FastBootRequest {
    hostWhitelist: [Array],
    protocol: 'undefined:',
    headers: [FastBootHeaders],
    queryParams: undefined,
    path: undefined,
    method: undefined,
    body: undefined,
    cookies: {} },
```

**Comment by ryanto** [comment]
Ok I think I follow and this should be a quick fix.

Could I see how your application is using the request path? I want to get some more test cases coded around the `FastBootRequest`.

**Comment by zeppelin** [comment]
I use `request.path` to determine the locale:

```js
let path = this.isFastBoot
  ? this.fastboot.get('request.path')
  : window.location.pathname;

let locale;
for (let availableLocale of availableLocales) {
  if (path.startsWith(`/${availableLocale}/`)) {
    locale = availableLocale;
    break;
  }
}

locale = locale || this.cookies.read('locale') || defaultLocale;
// ...
```

**Comment by ryanto** [comment]
Thanks! I see what this is. I tagged this as a bug. I'll update here when it's fixed.

**Comment by viniciussbs** [comment]
I haven't tried this addon yet but, just to give you more input, I use the request object to get the host.

```js
const host = this.get('fastboot.request.host');
```

**Comment by ryanto** [comment]
Hi @viniciussbs !

Thanks for letting me know! How are you using host in your application today? This addon currently supplies a hostname of `ember-cli-fastboot-testing.localhost`, would you like this to be configurable during the test?

**Comment by viniciussbs** [comment]
I need to set a request header with the host so the API I'm hitting knows what client it should fetch data from. It's how the API works with multitenancy. I guess I can rely on this fixed hostname, no problem.

**Comment by ryanto** [comment]
I think I have this covered in https://github.com/embermap/ember-cli-fastboot-testing/pull/11

`request.path` will be the url you're requesting from fastboot. It should be enough to not crash you app :)

**Comment by ryanto** [comment]
released as `0.0.5`
